connectors/cli.py

```diff
@@ -86,6 +97,12 @@ def _parser():
 
 
 async def _start_service(config, loop):
+    """Starts the service.
+
+    Steps:
+    - performs a preflight checks using `PreflightCheck`
```

Comment:
```suggestion
    - performs a preflight check using `PreflightCheck`
```

Reply:
I'm assuming it's one preflight check. If there are multiple then of course `performs preflight checks` would be better.

Reply:
we are doing several steps but I would assume that `preflight check` is fine -- devs can go and look at `PreflightCheck` I assume